content/ar/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html

```diff
@@ -0,0 +1,101 @@
+
+            <div class="col-md-8">
+                <h3>عناقيد كوبيرنيتيس</h3>
+                <p>
+                    <b>تنسق كوبيرنيتيس عنقودًا من الحواسيب المتاحة بشكل عالي والمتصلة لتعمل كوحدة واحدة.</b> التجريدات في كوبيرنيتيس تتيح لك نشر التطبيقات المعبأة في حاويات إلى عنقود دون ربطها بشكل خاص بالأجهزة الفردية. للاستفادة من هذا النموذج الجديد للنشر، يجب أن تكون التطبيقات معبأة بطريقة تفصلها عن المضيفين الفرديين: يجب أن تكون معبأة في حاويات. التطبيقات المعبأة في الحاويات أكثر مرونة وتوفرًا من النماذج السابقة للنشر، حيث كانت التطبيقات مثبتة مباشرة على أجهزة معينة كحزم متكاملة تكاملًا عميقًا مع المضيف. <b>كوبيرنيتيس يتمتع بالقدرة على توزيع وجدولة حاويات التطبيقات عبر العنقود بطريقة أكثر كفاءة.</b> كوبيرنيتيس هي منصة مفتوحة المصدر وجاهزة للإنتاج.
+                </p>
+                <p>
+                    يتكون عنقود كوبيرنيتيس من نوعين من الموارد:
+                    <ul>
+                        <li><b>مستوى التحكم</b> ينسق العنقود</li>
+                        <li><b>العقد</b> هي العمال الذين يقومون بتشغيل التطبيقات</li>
+                    </ul>
+                </p>
+            </div>
+
+            <div class="col-md-4">
+                <div class="content__box content__box_lined">
+                    <h3>الملخص:</h3>
+                    <ul>
+                        <li>عنقود كوبيرنيتيس</li>
+                        <li>مينيكيوب</li>
+                    </ul>
+                </div>
+                <div class="content__box content__box_fill">
+                    <p><i>
+                        كوبيرنيتيس هو منصة مفتوحة المصدر جاهزة للإنتاج تقوم بتنظيم وتنسيق وتنفيذ حاويات التطبيق داخل وعبر عناقيد الحواسيب.
+                    </i></p>
+                </div>
+            </div>
+            
+            <div class="row">
+                <div class="col-md-8">
+                    <h2 style="color: #3771e3;">مخطط العنقود</h2>
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="col-md-8">
+                    <p><img src="/docs/tutorials/kubernetes-basics/public/images/module_01_cluster.svg"></p>
+                </div>
+            </div>
+            <br>
+    
+            <div class="row">
+                <div class="col-md-8">
+                    <p><b>مستوى التحكم مسؤول عن إدارة العنقود.</b> يتولى مستوى التحكم جميع الأنشطة في عنقودك، مثل جدولة التطبيقات، والحفاظ على الحالة المطلوبة للتطبيقات، وتوسيع التطبيقات، ونشر التحديثات الجديدة.</p>
+                    <p><b>العقدة هي جهاز افتراضي أو كمبيوتر فعلي يعمل كجهاز عامل في عنقود كوبيرنيتيس.</b> كل عقدة تحتوي على kubelet، وهو وكيل لإدارة العقدة والتواصل مع مستوى التحكم في كوبيرنيتيس. يجب أن تحتوي العقدة أيضًا على أدوات لمعالجة عمليات الحاوية، مثل containerd أو cri-o. يجب أن يحتوي عنقود كوبيرنيتيس الذي يدير حركة الإنتاج على الأقل ثلاثة عقد لأنه إذا تعطل أحد العقد، يتم فقدان كل من عضو etcd ونسخة من مستوى التحكم، ويتعرض التكرار للخطر. يمكنك التخفيف من هذا المخاطر عن طريق إضافة المزيد من عقد مستوى التحكم.</p>
```

Comment:
I find the glossary tips useful you can also refer and try to push https://github.com/google/docsy/issues/1997
and https://kubernetes.slack.com/archives/C0191RDKHU1/p1715605527141379 you can reply to this and also sending it to the channel so it gets more views and ask how localization people are using this glossary tips currently 
```suggestion
                    <p><b>العقدة هي جهاز افتراضي أو كمبيوتر فعلي يعمل كجهاز عامل في عنقود كوبيرنيتيس.</b> كل عقدة تحتوي على kubelet، وهو وكيل لإدارة العقدة والتواصل مع مستوى التحكم في كوبيرنيتيس. يجب أن تحتوي العقدة أيضًا على أدوات لمعالجة عمليات الحاوية، مثل {{< glossary_tooltip text="containerd" term_id="containerd" >}}  أو cri-o. يجب أن يحتوي عنقود كوبيرنيتيس الذي يدير حركة الإنتاج على الأقل ثلاثة عقد لأنه إذا تعطل أحد العقد، يتم فقدان كل من عضو etcd ونسخة من مستوى التحكم، ويتعرض التكرار للخطر. يمكنك التخفيف من هذا المخاطر عن طريق إضافة المزيد من عقد مستوى التحكم.</p>
```

Reply:
تكرار وحدها غريبة https://ar.wikipedia.org/wiki/%D8%AA%D9%83%D8%B1%D8%A7%D8%B1_%D8%A7%D8%AD%D8%AA%D9%8A%D8%A7%D8%B7%D9%8A_(%D9%87%D9%86%D8%AF%D8%B3%D8%A9)